src/components/Navigator/NavigatorCard.vue

```diff
@@ -28,6 +33,18 @@
         @keydown.up.exact.capture.prevent="focusPrev"
         @keydown.down.exact.capture.prevent="focusNext"
       >
+        <Reference
```

Comment:
Just out of curiosity, what's the purpose of moving this from the Base component to here?

Reply:
Because I moved the `technology-title` element outside the `navigator-head` slot into the `body` sloth instead. Also it doesn't need to be shared with other components that use the `BaseNavigatorCard` as the `LoadingNavigatorCard`.